python/lsst/ts/standardscripts/maintel/laser_tracker/setup.py

```diff
@@ -0,0 +1,89 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["SetUp"]
+
+import asyncio
+import logging
+
+from lsst.ts import salobj
+from lsst.ts.observatory.control.maintel import MTCS
+
+
+class SetUp(salobj.BaseScript):
+    """
+    Set Up Laser Tracker component.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    None
+
+    """
+
+    def __init__(self, index):
+        super().__init__(index, descr="Set up laser tracker.")
+
+        self.config = None
+
+        self.laser_tracker = salobj.Remote(self.domain, name="LaserTracker", index=1)
+        self.mtcs = MTCS(domain=self.domain, log=self.log)
+
+        self.log.setLevel(logging.DEBUG)
+
+    @classmethod
+    def get_schema(cls):
+        return None
+
+    async def configure(self, config):
+        self.config = config
+
+    async def startup(self):
+        await self.laser_tracker.start_task(timeout=5)
+        await self.mtcs.start_task()
+
+        try:
+            await self.laser_tracker.evt_heartbeat.next(flush=True, timeout=5)
+        except asyncio.TimeoutError:
+            raise RuntimeError("Laser Tracker not responding to heartbeat.")
+
+        await self.mtcs.assert_liveliness()
+
+    async def run(self):
+        await self.startup()
+
+        await self.laser_tracker.cmd_enable.start(level=20)
+        await self.laser_tracker.cmd_setLogLevel.sstart(level=20)
```

Comment:
probably not a good idea to set the log level of the CSC in a Script. If someone wants to set the log level, they should do it themselves via LOVE.